src/mesh.cpp

```diff
@@ -2933,6 +2933,7 @@ LibMesh::LibMesh(pugi::xml_node node) : UnstructuredMesh(node)
 
 // create the mesh from a pointer to a libMesh Mesh
 LibMesh::LibMesh(libMesh::MeshBase& input_mesh, double length_multiplier)
+  : adaptive_(input_mesh.n_active_elem() != input_mesh.n_elem())
```

Comment:
I'm curious as to how robust this is. Are there other reasons that the number of active elements wouldn't equal the number of total elements in libMesh/MOOSE?

Reply:
As far as I am aware, active elements are only defined based on the current refinement / coarsening hierarchy in LibMesh and so on-the-fly h-adaptivity is the only system which causes this inequality to be true. Non-h-adaptive mesh operations (such as creating a submesh or deleting blocks) are treated as a post-procesing step (even when done online), modifying the total number of elements and active number of elements equally.

Reply:
Excellent. Thanks for verifying!